sources/packages/web/src/views/student/financial-aid-application/Applications.vue

```diff
@@ -50,7 +51,7 @@ export default {
       programYearList.value = programYears.map((programYear: ProgramYear) => ({
         name:
           "(" + programYear.programYear + ") - " + programYear.programYearDesc,
-        code: programYear.formName,
+        code: programYear,
```

Comment:
Please consider renaming the variable "code" to something like "programYear" since it no longer represents the code.